swap_meet/vendor.py

```diff
@@ -1,2 +1,73 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory= None):
+        if not inventory:
+            inventory = []
+        self.inventory = inventory
+
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return False
+
+
+    def get_by_category(self, category):
+        list_items = []
+        for item in self.inventory:
+            if item.category == category:
+                list_items.append(item)
+        return list_items
+
+
+    def swap_items(self, vendor, my_item, their_item):
+        if my_item not in self.inventory or their_item not in vendor.inventory:
+            return False
+        other_index = vendor.inventory.index(their_item)
+        self_index = self.inventory.index(my_item)
+        temp_item = my_item
+        self.inventory[self_index] = their_item
+        vendor.inventory[other_index] = temp_item
+        return True
+
+
+    def swap_first_item(self, vendor):
+        if self.inventory and vendor.inventory:
+            return self.swap_items(vendor, self.inventory[0], vendor.inventory[0])
```

Comment:
Very nice! Takes advantage of .swap_items() return value and guard clause will prevent an IndexError 👍🏻 Love it!